### PR TITLE
Add docs directory for RTD - again

### PR DESCRIPTION
**Pull request by pmatulis**
This PR adds a `docs` directory to the root of the repository. This is to have MicroOVN documentation published on readthedocs.com (RTD). The origin of this docs setup is the [sphinx-docs-starter-pack](https://github.com/canonical/sphinx-docs-starter-pack) project, which is what all RTD-based company projects are expected to use.

As agreed, all the stub pages have been removed. The landing page has just a few notes on next steps.

Doc versioning is possible but is currently disabled.

There is a link enabled in the footer for asking community questions in https://discourse.ubuntu.com/ but I recommend that we soon carve out a separate space (Discourse category) for MicroOVN on that forum - and change the link when that's done.

### CODE REVIEW

File: docs/.wordlist.txt
Comment (mkalcok) [inline docs/.wordlist.txt:1]: If this is a list of words for autocorrect, I'd suggest following
* ovsdb
* OVN
* ovnnb
* ovnsb
* northd
* openvswitch

File: docs/.github/workflows/automatic-doc-checks.yml
Comment (mkalcok) [inline docs/.github/workflows/automatic-doc-checks.yml:1]: I'd suggest removing this directory entirely. We have copy of this workflow file in root workflow directory (where it should be imo) and this additional copy could cause some confusion.

**Review by mkalcok — CHANGES_REQUESTED** [review]
Thank you for jump starting docs Peter. Just two minor suggestions below. Otherwise it LGTM.

File: docs/.github/workflows/automatic-doc-checks.yml
Comment (pmatulis) [reply docs/.github/workflows/automatic-doc-checks.yml:1]: Originally I wanted to use a symlink to the docs directory but I read that this wouldn't work with GitHub. I'll remove it.

File: docs/.wordlist.txt
Comment (pmatulis) [reply docs/.wordlist.txt:1]: I'll add these words.

**Review by mkalcok — APPROVED** [review]
LGTM. Thank you.

**Pull request merged**